lib/rgeo/geo_json/conversion_methods.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module RGeo
+  # This module serves to provide handy methods when using GeoJSON. The methods
+  # provided eases the passage between entities and GeoJSON String/Hash.
+  module GeoJSON::ConversionMethods
+    # Convert a geometry to a GeoJSON Hash
+    def as_geojson
+      GeoJSON.encode(self)
+    end
+    alias as_json as_geojson
+
+    # Convert a geometry to a GeoJSON String
+    def to_geojson
+      ::MultiJson.dump(as_geojson)
+    end
+    alias to_json to_geojson
+  end
+
+  module Feature
+    module Instance
+      # Convert a geometry to a GeoJSON Hash
+      def as_geojson
+        GeoJSON.encode(self)
+      end
+      alias as_json as_geojson
+
+      # Convert a geometry to a GeoJSON String
+      def to_geojson
+        ::MultiJson.dump(as_geojson)
+      end
+      alias as_json as_geojson
```

Comment:
Should this be `alias to_json to_geojson`?

Reply:
Yes it should be!